www/whiteboard/inner.js

```diff
@@ -474,6 +474,8 @@ define([
 
 
         framework.onEditableChange(function () {
+            $('#cp-app-whiteboard-canvas-area').show();
```

Comment:
We may have the same issue than for Rich Text documents ("pad"), but I didn't test